share/sql/share.go

```diff
@@ -160,14 +137,6 @@ func (m *mgr) Share(ctx context.Context, md *provider.ResourceInfo, g *collabora
 		ShareWith:         shareWith,
 		SharedWithIsGroup: g.Grantee.Type == provider.GranteeType_GRANTEE_TYPE_GROUP,
 	}
-	share.UIDOwner = conversions.FormatUserID(md.Owner)
```

Comment:
Why is this no longer needed here?

Reply:
wow, good catch! I think I wanted to copy but I cut instead probably